core/experiment-2/packages/admin/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "wc_admin",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "next build",
+    "dev": "rm -rf .next; NEXT_PRIVATE_LOCAL_WEBPACK=true next dev -p 3000",
```

Comment:
I'd like to learn. Can you share the logic for using NEXT_PRIVATE_LOCAL_WEBPACK=true ?

Reply:
sure

Reply:
NEXT_PUBLIC_LOCAL_: Environment variables with this prefix are visible on both the server and the client (browser)

NEXT_PRIVATE_LOCAL_: The environment variable with this prefix is only visible on the server and cannot be accessed by the client (browser). This is useful for storing sensitive information (such as database credentials, API keys for third-party services, etc.) as this information should only be used on the server and should not be exposed to the client.